tests/blackbox/stratisd_cert.py

```diff
@@ -58,6 +58,7 @@ def make_test_filesystem(pool_path, fs_name):
     return array_of_tuples_with_obj_paths_and_names[0][0]
 
 
+# pylint: disable=too-many-public-methods
```

Comment:
You only want to disable for this class, so put it on the same line as the ```class``` line, not above it.

Reply:
Resolved.